frontend/packages/api_client/src/models/DisplayStaticContentPages.ts

```diff
@@ -0,0 +1,97 @@
+/* tslint:disable */
+/* eslint-disable */
+/**
+ * OpenCraft Instance Manager
+ * API for OpenCraft Instance Manager
+ *
+ * The version of the OpenAPI document: api
+ * 
+ *
+ * NOTE: This class is auto generated by OpenAPI Generator (https://openapi-generator.tech).
+ * https://openapi-generator.tech
+ * Do not edit the class manually.
+ */
+
+import { exists, mapValues } from '../runtime';
+/**
+ * 
+ * @export
+ * @interface DisplayStaticContentPages
+ */
+export interface DisplayStaticContentPages {
+    /**
+     * 
+     * @type {boolean}
+     * @memberof DisplayStaticContentPages
+     */
+    about: boolean;
+    /**
+     * 
+     * @type {boolean}
+     * @memberof DisplayStaticContentPages
+     */
+    contact: boolean;
+    /**
+     * 
+     * @type {boolean}
+     * @memberof DisplayStaticContentPages
+     */
+    donate: boolean;
+    /**
+     * 
+     * @type {boolean}
+     * @memberof DisplayStaticContentPages
+     */
+    tos: boolean;
+    /**
+     * 
+     * @type {boolean}
+     * @memberof DisplayStaticContentPages
+     */
+    honor: boolean;
+    /**
+     * 
+     * @type {boolean}
+     * @memberof DisplayStaticContentPages
+     */
+    privacyPolicy: boolean;
+}
+
+export function DisplayStaticContentPagesFromJSON(json: any): DisplayStaticContentPages {
+    return DisplayStaticContentPagesFromJSONTyped(json, false);
+}
+
+export function DisplayStaticContentPagesFromJSONTyped(json: any, ignoreDiscriminator: boolean): DisplayStaticContentPages {
+    if ((json === undefined) || (json === null)) {
+        return json;
+    }
+    return {
+        
+        'about': json['about'],
+        'contact': json['contact'],
+        'donate': json['donate'],
+        'tos': json['tos'],
+        'honor': json['honor'],
+        'privacyPolicy': json['privacy_policy'],
+    };
+}
+
+export function DisplayStaticContentPagesToJSON(value?: DisplayStaticContentPages | null): any {
+    if (value === undefined) {
+        return undefined;
+    }
+    if (value === null) {
+        return null;
+    }
+    return {
+        
+        'about': value.about,
+        'contact': value.contact,
+        'donate': value.donate,
+        'tos': value.tos,
+        'honor': value.honor,
+        'privacy_policy': value.privacyPolicy,
+    };
```

Comment:
This section of the code seems to be repetitive between `DisplayStaticContentPagesToJSON` and `DisplayStaticContentPagesFromJSONTyped`, do you think it would be a good idea to group these return statements into a single function and return that function instead?

That way, if any change needs to be done to the json that is being returned, we won't have to do the change in two separate places.

Reply:
I am aware that the values are being retrieved differently,`value.about` and `json['about']`, but it should be possible to combine these 2 into a single function. What do you think?

Reply:
@nizarmah this is auto generated file (so we can skip `api_client` related changes). This is a client which was generated via openapi generator.

We can't modify it (it will be overridden after the next command call)

Reply:
Ah! Alright, thanks for clarifying that :D I'll remove all comments related to the auto-generated code. Can you please specify which files are auto generated and have a comment on them? Or just give me the format to find other auto generated files.

I'm guessing `frontend/packages/api_client/src/models/*`?

Reply:
Yep, that's right, everything inside `frontend/packages/api_client` is auto-generated code.

Reply:
I will resolve all related comments.

Reply:
Thanks a lot :)